handlers_test.go

```diff
@@ -32,3 +43,59 @@ func TestHealthEndpoint(t *testing.T) {
 		}
 	})
 }
+
+func TestPostUser(t *testing.T) {
+	dbURL := os.Getenv("PG_CONN")
+
+	db, err := sql.Open("postgres", dbURL)
+
+	if err != nil {
+		t.Errorf("can not open database connection")
+	}
+
+	provider, err := goose.NewProvider(
+		goose.DialectPostgres,
+		db,
+		os.DirFS("./sql/schema/"),
+	)
+
+	if err != nil {
+		t.Errorf("can not create goose provider %q", err)
+		return
+	}
+
+	results, err := provider.Up(context.Background())
+
+	if err != nil {
+		t.Errorf("could not run migrations %q", err)
+	}
+
+	t.Log(results)
+
+	dbQueries := database.New(db)
```

Comment:
I need a better way to 

1. Open a connection to the database
2. Migrate the schema to the latest version
3. Run tests that have a side effect of altering database state
4. Clean all tables
5. Run next test with clean state

Reply:
Done 😎